lib/index.js

```diff
@@ -117,6 +117,14 @@ GitIssues.prototype.fromProvider = function (providerName, user, repo, status, a
  */
 GitIssues.prototype.fromPath = function (path, status, auth, callback) {
 
+    try {
+      parsedUrl = GitUrlParse(path);
```

Comment:
This will never throw an error. To check if remote, you have to check if `parsedUrl.protocol !== "file"`. But I already fixed this. :balloon: